projects/lexical_shared/test/lexical/formats_test.exs

```diff
@@ -15,6 +15,14 @@ defmodule Lexical.FormatsTest do
     test "it correctly handles an erlang module name" do
       assert ":ets" == Formats.module(:ets)
     end
+
+    test "it drops any `Elixir.` prefix" do
+      assert "Kernel.SpecialForms" == Formats.module(Elixir.Kernel.SpecialForms)
+    end
```

Comment:
FYI, @Moosieus this this is basically the same thing as the above tests `Elixir.Kernel.SpecialForms` -> `Kernel.SpecialForms`